install/kots/Makefile

```diff
@@ -0,0 +1,20 @@
+CHANNEL_STABLE = Stable
+CHANNEL_BETA = Beta
+CHANNEL_UNSTABLE = Unstable
+YAML_DIR = manifests
+
+all: helm lint create_unstable_release
+
+create_unstable_release:
+	replicated release create --lint --yaml-dir ${YAML_DIR} --promote ${CHANNEL_UNSTABLE}
+.PHONY: create_unstable_release
+
+lint:
+	replicated release lint --yaml-dir ${YAML_DIR}
+.PHONY: lint
+
+helm:
+	@echo "Installing Helm dependencies"
+	@rm -f manifests/*.tgz
+	@for f in $(shell ls -d charts/*/); do cd $${f} && helm dep up && helm package . --destination ../../manifests && cd -; done
```

Comment:
Instead of this helm magic, couldn't we just download the cert-manager package like this?

```
$ curl -sSLO https://charts.jetstack.io/charts/cert-manager-v1.7.0.tgz
```

Or wouldn't this work for some reason?

Reply:
That would work, but I've done it as a general function in case we ever put a second Helm chart in there - we use `external-dns` in all the guides which might be worth offering in Kots in future

Reply:
Right, I think that my original approach is the correct one. Using the `.tgz` file provided by cert-manager has everything nested in a `cert-manager` folder. For KOTS to work, it needs to be in the root of the `.tgz` file.

So, that makes there two ways of achieving this:
1. create a noddy `Chart.yaml` file and use the `helm package` function
2. download the `.tgz` file, extract it, re-compress it with the `cert-manager` and use that new `.tgz` file

Whilst both have their limitations, I think approach `1` is the simpler and more maintainable one